easybuild/tools/__init__.py

```diff
@@ -38,3 +38,11 @@
 
 
 from easybuild.tools.loose_version import LooseVersion  # noqa(F401)
+
+
+def dump_toml(o: dict) -> str:
```

Comment:
Please move this to `easybuild.tools.filetools`, I strongly prefer keeping `easybuild.tools` largely empty as it is now